starter-code/src/App.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from "react";
+
+import "./App.css";
+
+// import "bootstrap/dist/css/bootstrap.css";
+
+class App extends Component {
+  render() {
+    // render() returns the html of this component's content
+    // (use parenthesis when returning multiple lines of html)
+    return (
+      // use "className" instead of "class" for styling
+      <div className="App container">
+        <div className="section1">
+          <div className="nav">
```

Comment:
Here, you could have used the tag nav and targeted in your css without adding a class name